qemu/tests/cfg/balloon_boot_in_pause.cfg

```diff
@@ -0,0 +1,34 @@
+- balloon_boot_in_pause: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
+    type = balloon_boot_in_pause
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    ratio_1 = 0.8
+    ratio_2 = 0.5
+    test_tags = "evict enlarge evict"
+    balloon_type_evict = evict
+    balloon_type_enlarge = enlarge
+    paused_after_start_vm = yes
+    variants:
+        - @balloon-base:
+
+        - balloon-migrate:
+            sub_test_after_balloon = "migration"
+            migration_test_command = help
+        - balloon-reboot:
+            sub_test_after_balloon = "boot"
+            reboot_method = shell
+            kill_vm_on_error = yes
+            reboot = yes
```

Comment:
@CongLi I believe it's useless as that in file cfg/balloon_check.cfg, will delete it. Thanks a lot.